webapp/src/app/admin/admin.component.ts

```diff
@@ -1,6 +1,6 @@
 import { Component } from "@angular/core";
 
-import Log from "@dedis/cothority/log";
+import Log from "src/lib/cothority/log";
```

Comment:
Ho yep, there is a __bunch__ of thoses lying around (52 with a rapid `grep | wc`).

Reply:
forgot `make npm` in there...